src/proto/ipc/v1/ipc.proto

```diff
@@ -4,76 +4,12 @@ package ipc.v1;
 
 service ExtensionService {
     rpc GetWorkspaceAuthInfo (GetWorkspaceAuthInfoRequest) returns (GetWorkspaceAuthInfoResponse) {}
-    rpc SendLocalSSHUserFlowStatus (SendLocalSSHUserFlowStatusRequest) returns (SendLocalSSHUserFlowStatusResponse) {}
```

Comment:
felling uneasy about it, since upgrade happen between any 2 version of extensions, but unimplemented result should not break a flow though?